src/content/1.7/basics/installation/distribution.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Create your own distribution
```

Comment:
```suggestion
title: Create release
```

Shouldn't be something more generic? ping @matks

Reply:
I disagree. Release is not what I talk. Distribution is the subject.

Reply:
I link to the build page for not duplicating some informations about creating a release.